src/main/resources/templates/user.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeLeaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>User Details</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM"
+          crossorigin="anonymous">
+</head>
+<body>
+<div align="center">
+    <h1>Add User</h1>
+    <br />
+    <form th:action="@{/user}" th:object="${user}" method="POST">
+        <table border="0" cellpadding="10">
+            <tr>
+                <td>Name:</td>
+                <td><input type="text" name="name" th:field="*{name}" placeholder="Enter Name" class="form-control"/></td>
+            </tr>
+            <tr>
+                <td>Phone:</td>
+                <td><input type="text" name="name" th:field="*{phone}" placeholder="Enter Phone" class="form-control"/></td>
+            </tr>
+            <tr>
+                <td>Email:</td>
+                <td><input type="text" name="name" th:field="*{email}" placeholder="Enter Email" class="form-control"/></td>
```

Comment:
Can we give correct `type` so that user is only able to input valid Email ID in this input field?